README.md

```diff
@@ -6,5 +6,16 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a framework that is written in Ruby that provides an organized approach to
```

Comment:
And additionally, it makes it easy to spin up a new back-end very quickly, since it gives us a lot of structure to start with.